lib/ts/recipe/passwordless/utils.ts

```diff
@@ -184,3 +186,151 @@ export async function setLoginAttemptInfo(input: {
         attemptInfo: input.attemptInfo,
     });
 }
+
+/**
+ * These functions are helper functions so that the logic can be exposed from both
+ * passwordless and thirdpartypasswordless recipes without having to duplicate code
+ */
+
+export async function createCode(
```

Comment:
Refactored passwordless/utils to use the util functions from web-js (I prefer that instead of having the index files use it directly). Note that the import is currently using `supertokens-web-js/lib/build`, all `lib/build` imports will get cleaned up in another PR